fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java

```diff
@@ -358,11 +466,18 @@ private boolean relocateAndBalance(GroupId groupId, Set<Long> unavailableBeIds,
                 }
             }
 
+            if (!disableRepairPrecedence && !hasUnavailableBe && !unavailableBeIds.isEmpty() && isChanged) {
```

Comment:
How about this case?
The table has one replica and failed. 
The repair can not succeed but occupy the slot. The normal balance tasks can not advance.

Reply:
This pr https://github.com/StarRocks/starrocks/pull/11730 optimize single replica balance. It will not cause relocation if there is only one replica.